src/commands/pe-symbols/README.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+
+Upload Windows PE debug info files to Datadog to symbolicate your profiles.
+
+
+## Setup
+
+You need to have `DATADOG_API_KEY` in your environment.
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+You can configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable as `datadoghq.eu`. By default, the requests are sent to Datadog US.
```

Comment:
Can customers use the other sites too, or is it just US1 and EU?

Reply:
This is the same wording as [3 other commands](https://github.com/search?q=repo%3ADataDog%2Fdatadog-ci%20%22You%20can%20configure%20the%20tool%20to%20use%20Datadog%20EU%20by%20defining%22&type=code), and I agree it's misleading. The point we want to make is that by default, the requests are sent to US1. And to change that, you can set the `DATADOG_SITE` to anything [in this table](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site).

I'm already planning to change this globally in [this branch](https://github.com/DataDog/datadog-ci/compare/corentin.girard/SYNTH-11863/inconsistent-use-of-env-vars): [link to a modified README](https://github.com/DataDog/datadog-ci/blob/corentin.girard/SYNTH-11863/inconsistent-use-of-env-vars/src/commands/elf-symbols/README.md).